azext_iot/central/services/api_token.py

```diff
@@ -45,14 +46,22 @@ def add_api_token(
         "roles": [{"role": role.value}],
     }
 
+    # Construct parameters
+    query_parameters = {}
```

Comment:
There is quite a bit of repeated code blocks defining query_parameters (in this module and other modules in the service layer). Because this layer is not currently generated code, it may make sense to build some kind of wrapper that applies `api-version` (and future query string params) in a consistent way.

Reply:
This same wrapper could apply `verify=not should_disable_connection_verify()` consistently across the board.

Reply:
wont fix:  Great suggestion.  Work to use autorest API's is in the backlog. this will be addressed with the new model.